br/pkg/restore/rawkv_client.go

```diff
@@ -67,12 +78,29 @@ func (c *RawKVBatchClient) SetColumnFamily(columnFamily string) {
 
 // Put puts (key, value) into buffer justly, wait for batch write if the buffer is full.
 func (c *RawKVBatchClient) Put(ctx context.Context, key, value []byte) error {
-	c.keys = append(c.keys, key)
-	c.values = append(c.values, value)
-	c.size++
+	k, ts, err := SplitKeyTS(key)
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	sk := bytes2String(k)
+	if v, ok := c.kvs[sk]; ok {
+		if v.ts < ts {
```

Comment:
```suggestion
		if v.ts <= ts {
```

Reply:
it has no difference

Reply:
OK

Reply:
> it has no difference

What if we put two different value with the same key? (Note that we have rewritten all write cf keys into the same TS)

Reply:
Do you mean same key and same ts,  but different value?

Reply:
This situation for the same key, the latter KV entry will overwrite the former. 
There is no problem.

Reply:
em... this should be an issue that always exists

Reply:
but the order in file are not guaranteed, cause we have incremental scan and cmd from observer.

Reply:
> Do you mean same key and same ts, but different value?

Yes, that is possible because we have rewritten all write records to the same ts.

Reply:
For different key, the order can not be guaranteed, And the same Key, the order can be guaranteed because of the scan operation with the key ascending order.